docs/MAINTAINERS.md

```diff
@@ -2,4 +2,4 @@
 
 Vadim Markovtsev [vadim@sourced.tech](mailto:vadim@sourced.tech) \(@vmarkovtsev\)
 
-Guillem Duran [guillem@sourced.tech](mailto:vadim@sourced.tech) \(@guillemdb)
\ No newline at end of file
+Guillem Duran [guillem@sourced.tech](mailto:vadim@sourced.tech) \(@guillemdb\)
```

Comment:
I missed the MAINTAINERS file yesterday, it should stay top-level according to our company-wide guidelines.